### PR TITLE
feat: Create quick chat discussion when no filter results - EXO-60916

### PR DESCRIPTION
**Pull request by sofyenne**
We will create a quick chat discussion when searching a discussion has no results.

### CODE REVIEW

File: application/src/main/webapp/css/components/chatContact.less [hunk 1]
```diff
@@ -110,4 +110,18 @@
     font-weight: bold;
   }
 }
+.noResultFilter {
+  position: absolute;
+  top: 50%; left: 50%; 
+  transform: translate(-50%, -50%);
+  width:350px
+}
+#noresultFiltertmessage{
```
Comment (azayati) [inline application/src/main/webapp/css/components/chatContact.less]:
```suggestion
#noResultFiltertmessage {
```

File: application/src/main/webapp/css/components/chatContact.less [hunk 2]
```diff
@@ -110,4 +110,18 @@
...
+  color: @grayDark;
+}
+#noresultFilterPlusIcon{
```
Comment (azayati) [inline application/src/main/webapp/css/components/chatContact.less]:
```suggestion
#noResultFilterPlusIcon {
```

File: application/src/main/webapp/css/components/chatContact.less [hunk 3]
```diff
@@ -110,4 +110,18 @@
...
+  color: #4d5466;
```
Comment (azayati) [inline application/src/main/webapp/css/components/chatContact.less]: use color code

File: application/src/main/webapp/css/components/chatContact.less [hunk 4]
```diff
@@ -110,4 +110,18 @@
...
+}
+
+
```
Comment (azayati) [inline application/src/main/webapp/css/components/chatContact.less]: remove empty lines

File: application/src/main/webapp/vue-app/components/ExoChatContactList.vue [hunk 5]
```diff
@@ -768,7 +788,10 @@ export default {
         return chatServices.getSpaceProfileLink(this.contactMenu.fullName);
       }
       return '#';
-    }
+    },
+    openQuickCreateChatDiscussionDrawer(){
```
Comment (azayati) [inline application/src/main/webapp/vue-app/components/ExoChatContactList.vue]:
```suggestion
    openQuickCreateChatDiscussionDrawer() {
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: application/src/main/webapp/vue-app/components/ExoChatContactList.vue (hunk 5 above)
Comment (sofyenne) [reply application/src/main/webapp/vue-app/components/ExoChatContactList.vue]: done !

File: application/src/main/webapp/css/components/chatContact.less (hunk 4 above)
Comment (sofyenne) [reply application/src/main/webapp/css/components/chatContact.less]: done !

File: application/src/main/webapp/css/components/chatContact.less (hunk 3 above)
Comment (sofyenne) [reply application/src/main/webapp/css/components/chatContact.less]: done !

File: application/src/main/webapp/css/components/chatContact.less (hunk 2 above)
Comment (sofyenne) [reply application/src/main/webapp/css/components/chatContact.less]: done !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**